go/client/cmd_show_notifications.go

```diff
@@ -99,6 +99,9 @@ func (d *notificationDisplay) LoggedOut(_ context.Context) error {
 func (d *notificationDisplay) LoggedIn(_ context.Context, un string) error {
 	return d.printf("Logged in as %q\n", un)
 }
+func (d *notificationDisplay) ClientOutOfDate(_ context.Context, arg keybase1.ClientOutOfDateArg) error {
+	return d.printf("Client out of date, upgrade to \"%s\", uri \"%s\"\n", arg.UpgradeTo, arg.UpgradeURI)
+}
```

Comment:
@cjb did you mean this notification text? This command is just for testing. You run `keybase show-notifications` and it spins waiting for notifications to happen. This PR doesn't include any actual user-facing UI. I thought we might want to show the existing "Please Upgrade" window, maybe with some extra text inserted in there to make it clear that we Mean Business This Time?